beprof/profile.py

```diff
@@ -144,11 +146,14 @@ def normalize(self, dt):
         except RuntimeWarning as e:
             logger.error('in normalize(). self class is {0}, dt={1}'.format(self.__class__, dt))
             raise Exception("Scaling factor error:\n" + str(e))
-        self.y /= ave
+        self.y = self.y / ave
```

Comment:
Second one better, but please add also proper docstring with doctests (see i.e. change_domain and rebinned methods).

Read also this: http://www.ianbicking.org/blog/2007/09/re-raising-exceptions.html
https://blog.ionelmc.ro/2014/08/03/the-most-underrated-feature-in-python-3/